src/utils/session-helper.js

```diff
@@ -249,3 +251,36 @@ export const updateTTLFile = async (session, containerUrl, fileObject) => {
     throw new Error('Failed to update ttl file.');
   }
 };
+
+
+
+/**
+ * Creates a TTL file corresponding to an uploaded document or resource
+ *
+ * @memberof utils
+ * @function createResourceTtlFile
+ * @param {fileObjectType} fileObject - Object containing information about file
+ * from form submission (see {@link fileObjectType})
+ * @param {documentUrl} String - url of uploaded document or resource
```

Comment:
For Line 264, we can fix the JSDoc error:

```JSDoc
@param {URL} documentUrl
```

JSDoc uses the format of:
```
@param {<type>} <variableName>
```

Reply:
Ah ok that's how jsdoc works. That makes more sense. Still trying to learn it.